src/Shared/Recruit.Vacancies.Client/Application/Validation/Fluent/FluentVacancyValidator.cs

```diff
@@ -135,7 +135,37 @@ private void ValidateOrganisation()
                     .RunCondition(VacancyRuleSet.TradingName)
                     .WithRuleId(VacancyRuleSet.TradingName)
             );
-            
+
+            When(v => v.SourceOrigin == SourceOrigin.EmployerWeb && v.EmployerNameOption == EmployerNameOption.Anonymous, () =>
+                RuleFor(x => x.EmployerName)
+                    .NotEmpty()
+                    .WithMessage("You must provide a description")
```

Comment:
description?

Reply:
Its in the story. Employer name is a description of what the employer does not an alias.